models/sdn-fabric-0.1.x/openapi.yaml

```diff
@@ -167,6 +167,22 @@ components:
       x-keys:
       - dhcp-server-id
       x-list-multiple: true
+    LeafRefResolver:
```

Comment:
looks good - a better name might be `LeafRefOptions`

Reply:
@SeanCondon Yes, I will create a separate PR to update the models. sd-fabric here is just a temporary example.